docs/source/quickstart/01_requirements.md

```diff
@@ -1,10 +1,39 @@
-# Minimum requirements
+# System requirements
 
-To run the ci-framework locally on your machine you will need:
+You usually want to run against a remote hypervisor in order to get a working infrastructure.
 
-* 32 GB RAM
-* 120 GB free storage
-* 6 CPU
+## Lightweight deployment
```

Comment:
I would put deploy times here. Since we are already discussing the minimum reqs it would be nice to know what you get with them.

Reply:
indeed. moving block + split.